Source/Atomic/Network/Connection.cpp

```diff
@@ -200,6 +205,8 @@ void Connection::SetScene(Scene* newScene)
         scene_->StopAsyncLoading();
         SubscribeToEvent(scene_, E_ASYNCLOADFINISHED, HANDLER(Connection, HandleAsyncLoadFinished));
     }
+
+    SubscribeToEvent(scene_, E_COMPONENTREMOVED, HANDLER(Connection, HandleComponentRemoved));

```

Comment:
This seems like something that we may want to pass upstream to Urho3D?  I am surprised we needed to add it, seems core.  Does stock Urho3D leak component states without this?